wasm/zlib/src/main.c

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stdio.h>
+#include "zlib.h"
+
+// To be compiled together with `zpipe.c` from zlib. Forward declarations:
+int def(FILE *source, FILE *dest, int level);
+int inf(FILE *source, FILE *dest);
+void zerr(int ret);
+
+// Exported to JavaScript:
+int compressFile(const char* infilename, const char* outfilename)
+{
+    FILE* infile = fopen(infilename, "rb");
```

Comment:
Don't you need to `fclose` these? I assume this will leak memory in the test

Reply:
Oops, indeed. Thanks!